### PR TITLE
Trim URL so that whitespace at beginning or end won't reject it

### PR DESCRIPTION
**Pull request by gotonode**
Implements #946.

### CODE REVIEW

**Comment by gotonode** [comment]
This could be considered a bug or just a feature. Depends.

**Comment by gotonode** [comment]
Scheduled for next week so hours are divided evenly. I'll also check other such places for similar "errors". This is an accessibility issue.

**Comment by hisahi** [comment]
This is an issue at least when
* student submits a project name or link
* student submits a code review link

**Comment by gotonode** [comment]
Working on it right now.

**Comment by gotonode** [comment]
I also implemented a check to verify that the "courses.helsinki.fi" URL actually contains that part.

Also changed those fields from text to URL.

File: labtool2.0/src/components/pages/ReviewStudentCodeReview.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ import Error from '../Error'
 import { Points } from '../Points'
 import { roundPoints } from '../../util/format'
 
+// Changes go here, possibly.
```
Comment (hisahi) [inline labtool2.0/src/components/pages/ReviewStudentCodeReview.js]: not needed anymore, I suppose?

Comment (gotonode) [reply labtool2.0/src/components/pages/ReviewStudentCodeReview.js]: Yes, since this is done. Removed.

**Pull request merged**